### ISSUE TITLE
[View] `to-image` can't shoot an image face

### ISSUE THREAD

**Issue by hiiamboris** [red/red#3675]
**Describe the bug**

? (to-image view/no-wait [image 50x50])
![](https://i.gyazo.com/f8f88a057aca1cffe4b298b634783bfa.png)
(below is the to-image result)

**Platform version (please complete the following information)**
```
Red 0.6.4 for Windows built 18-Dec-2018/23:37:37+03:00 commit #873a756
W7 x64
```

**Comment by meijeru** [comment]
Works on W10

**Comment by hiiamboris** [comment]
Yes it is most likely W7-specific bug.

**Comment by qtxie** [comment]
Was it fixed by this PR #3683?

**Comment by hiiamboris** [comment]
Yeah, it works for me now.
